src/Constraint/MultiConstraint.php

```diff
@@ -86,7 +86,7 @@ public function matches(ConstraintInterface $provider)
     }
 
     /**
-     * @param string $prettyString
+     * @param string|null $prettyString
```

Comment:
This is weird, seems wrong, as protected $prettyString is typed `string`

Reply:
Relates to https://github.com/composer/semver/blob/39da74948fd424cf0e8321b96f04c2a1b2480a15/tests/Constraint/MultiConstraintTest.php#L91

Reply:
Right but then the property type is wrong, my point was just they are not consistent with each other so something is off.